### ISSUE TITLE
Doesn't work with 2023.4.1

### ISSUE THREAD

**Issue by ASNNetworks** [elchininet/lovelace-autoscroll#6]
Great module! I used 2.2.0 yesterday with 2023.3.6 (since 3.0 required 2023.4) and that worked great. Now I updated to 2023.4.1 and updated this module to 3.0, but it doesn't work anymore. The card seems to be loaded fine:

![image](https://user-images.githubusercontent.com/54230480/230616420-6adb24cf-79a1-43a2-90e2-d52f52008a64.png)

Tried re-installing the module multiple times, many cache clearing and restarting HA to be safe, but doesn't seem to work anymore.

Setup in lovelace.yaml is fine too:

```
autoscroll:
  behavior: smooth
  history_keep_scroll: false
```

**Comment by elchininet** [comment]
Hi @ASNNetworks,
I released `3.0.0` for `2023.4.0`, let me check if something gets broken in `2023.4.1`.
Regards

**Comment by elchininet** [comment]
Hi @ASNNetworks,
As I suspected, just from `2023.4.0` to `2023.4.1` they changed again the main element that scrolls the page.
But I am noticing something, it seems that they fixed one of the issues that this plugin tried to solve. When I navigate to a subview, the scroll is not maintained anymore in the same position, the subview is rendered with the scroll on the top. The only issue that is not fixed yet is keeping the same scroll position when navigating back through history.
I think that if they fixed the main issue with the scroll, this plugin is not really needed anymore.
Regards

**Comment by elchininet** [comment]
@ASNNetworks,
It is really annoying. I uninstalled autoscroll because it was working well yesterday, today, sometimes it works and sometimes it doesn't. I don't find any pattern or logic there. 🤔

**Comment by ASNNetworks** [comment]
@elchininet yes, I am very confused as well. I removed auto-scroll since it didn't work anyway with 2023.4.1, then after a few hours suddenly all my dashboards do start at the top, when switching back and forth. Without having auto-scroll even loaded or installed. 

There are definitely lots of weird things happening with frontend and how different custom modules now work with eachother since 2023.4.
